### PR TITLE
Added mail addresses to drek positions in descriptions

### PR DESCRIPTION
**Pull request by NilssonWilliam**

### CODE REVIEW

**Comment by elmaxe** [comment]
#181

File: organisation/sammansattning/body.md [hunk 1]
```diff
@@ -168,3 +178,5 @@ __Kontaktfunktionärer__
 * Sektionshistoriker
 
 * Chefredaqtör
+
+Ledamot för näringsliv och kommunikation frågor är nåbar på d-nok@d.kth.se
```
Comment (Grunler) [inline organisation/sammansattning/body.md:182]:
```suggestion
Ledamot för näringsliv och kommunikation är nåbar på d-nok@d.kth.se
```

**Review by Grunler — CHANGES_REQUESTED** [review]

**Review by MalinSvenberg — APPROVED** [review]
Sgtm 😊

**Review by Grunler — APPROVED** [review]

**Comment by Grunler** [comment]
lgtm

**Comment by oskstr** [comment]
Shouldn't the email addresses perhaps have the format: 
```markdown
[ordf@d.kth.se](mailto:ordf@d.kth.se)
```
to be a link that opens in an email client like we have in other places? like here [kontakt/body.md](https://github.com/datasektionen/bawang-content/blob/master/kontakt/body.md)

**Pull request merged**